src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,508 @@
+they should be added to the `*recording-blood_discrete.tsv`
+and the values below should be added to the `*recording-blood_discrete.json`.
+If whole blood measurements are available by a continuous sampler, for example, using an Allogg autosampler,
+they should be added to the `*recording-blood_continous.tsv`
+and the values below should be added to the `*recording-blood_continous.json`.
+
+| Field name  | Definition                                                                                  |
+|:------------|:--------------------------------------------------------------------------------------------|
+| ColumnName  | REQUIRED. Name of the column (for example, "whole_blood_radioactivity)".                     |
+| Description | REQUIRED. Radioactivity in whole blood samples.                                             |
+| Units       | REQUIRED. Unit of radioactivity measurements in whole blood samples (for example, "kBq/mL") |
+
+### Example (`*_blood.json`)
+
+```JSON
+{
+	"PlasmaAvail": true,
+	"MetaboliteAvail": true,
+	"ContinuousBloodAvail": true,
+	"DiscreteBloodAvail": true
+}
+```
+
+### Example (`*_recording-blood_discrete.json`)
+
+```JSON
+{
+    "time": {
+        "Description": "Time in relation to time zero defined by the _pet.json",
+        "Units": "s"
+    },
+    "plasma_radioactivity": {
+        "Description": "Radioactivity in plasma samples. Measured using COBRA counter.",
+        "Units": "kBq/mL"
+    },
+    "whole_blood_radioactivity": {
+        "Description": "Radioactivity in whole blood samples. Measured using COBRA counter.",
+        "Units": "kBq/mL"
+    },
+    "metabolite_parent_fraction": {
+        "Description": "Parent fraction of the radiotracer.",
+        "Units": "arbitrary"
+    },
+    "metabolite_polar_fraction": {
+        "Description": "Polar metabolite fraction of the radiotracer.",
+        "Units": "arbitrary"
+    },
+    "metabolite_lipophilic_fraction": {
+        "Description": "Lipophilic metabolite fraction of the radiotracer.",
+        "Units": "arbitrary"
+    }
+}
+```
+
+![Figure 2](images/blood_discrete_tsv.png "Screenshot of the discrete blood tsv file.")
+
+Figure 2: Caption of the corresponding `recording-blood_discrete.tsv` file.
+
+
+### Example (`*_recording-blood_continuous.json`)
+
+```JSON
+{
+"time": {
+        "Description": "Time in relation to time zero defined by the _pet.json",
+        "Units": "s"
+    },
+    "plasma_radioactivity": {
+        "Description": "Radioactivity in plasma samples. Measured using Allogg autosampler.",
+        "Units": "kBq/mL"
+    }
+}
+```
+
+# Appendix: Useful resources
+
+This website is an extremely useful resource: http://www.turkupetcentre.net/petanalysis/quantification.html
+
+An extensive resource for background knowledge on kinetic modeling
+can be found here: https://nru.dk/index.php/misc/category/3-pet-kinetic-course
```

Comment:
I suggest to remove this sub paragraph --> the links to resource should be woven in with with the specification text where it's relevant. And if there is no such text in the specification, the recommended links could be placed in the intro to PET.

Reply:
This has been resolved and we have made a regular Appendix for PET.